_episodes/028-twir-352.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "This Week in Rust - Issue 352"
+date: 2020-08-12T14:30:00Z
```

Comment:
This date doesn't seem right?

Reply:
Oh! Fixing!

Reply:
Corrected - though I have back dated it to the 18th, as that's when I said it was published in the podcast

Reply:
Hope that is ok, if not I can change it.

Reply:
Oh, yeah, that's fine.